<launch>

<include file="$(find color_mask)/launch/blue_mask_zero.launch"/>
<include file="$(find object_centroid)/launch/blue_centroid_zero.launch"/>
<include file="$(find object_slope)/launch/blue_slope_zero.launch"/>
<include file="$(find pid)/launch/pid_slope_zero.launch"/>

<include file="$(find color_mask)/launch/blue_mask_one.launch"/>
<include file="$(find object_centroid)/launch/blue_centroid_one.launch"/>
<include file="$(find object_slope)/launch/blue_slope_one.launch"/>
<include file="$(find pid)/launch/pid_slope_one.launch"/>


<include file="$(find aruco_detector)/launch/aruco_detector_zero.launch"/>
<include file="$(find object_distance)/launch/aruco_distance_zero.launch"/>
<include file="$(find pid)/launch/pid_distance_zero.launch"/>

<include file="$(find aruco_detector)/launch/aruco_detector_one.launch"/>
<include file="$(find object_distance)/launch/aruco_distance_one.launch"/>
<include file="$(find pid)/launch/pid_distance_one.launch"/>


  <group ns="zero">
      <node pkg="jetbot_demos" name="lane_following_demo_1_node" type="lane_following_demo_1.py">
          <param name="MAX_VELOCITY" value="0.28"/>
          <param name="MIN_VELOCITY" value="0.26"/>
          <param name="SHARP_TURN" value="0.1"/>
          <param name="CONTROLLER_READY" value="0"/>
      </node>
  </group>

  <group ns="one">
      <node pkg="jetbot_demos" name="lane_following_demo_1_node" type="lane_following_demo_1.py">
          <param name="MAX_VELOCITY" value="0.28"/>
          <param name="MIN_VELOCITY" value="0.26"/>
          <param name="SHARP_TURN" value="0.1"/>
          <param name="CONTROLLER_READY" value="0"/>
      </node>
  </group>

</launch>
